src/keeper/contracts/ContractBase.ts

```diff
@@ -119,6 +119,10 @@ export abstract class ContractBase extends Instantiable {
 
             if (value) gas += 21500
```

Comment:
shouldn't we remove the `value` parameter?
Or maybe even convert that `value` parameter in the multiplier parameter (0% booster by default). When this is given by the method integrating the `send` method expedites that specific transaction.

Reply:
We probably cannot remove it because it's needed for fulfilling conditions with Eth payment. I'll check the tx parameters, hopefully it's easy to add a new parameter instead...

Reply:
I see, so it's used in `payable` calls. In that case you are right, the best approach would be to add the `gasMultiplier` as a parameter with a default value that implies no gas "booster"